rules/no-skip-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+module.exports = function (context) {
+	var ava = createAvaRule();
+	return ava.merge({
+		MemberExpression: function (node) {
+			if (ava.isTestFile && node.property.name === 'skip' && util.isCallFromTestObject(node)) {
```

Comment:
Couldn't this use `ava.hasTestModifier('skip')`?

https://github.com/sindresorhus/eslint-plugin-ava/pull/20/files#diff-a683eee4dfe24b3d7070e7482ded2411R10